configure.ac

```diff
@@ -216,6 +216,23 @@ AC_LINK_IFELSE(
 
 AM_CONDITIONAL([WITH_PCAP_FIX], [test "$pcap_fix" = "yes"])
 
+# Check for protobuf compiler
+AC_PATH_PROG([PROTOC], [protoc], [])
+AS_IF([test "x$PROTOC" = x], [AC_MSG_ERROR([protoc not found])])
+
+# Check for gRPC protocol buffers compiler plugins
+AC_PATH_PROG([GRPC_CPP_PLUGIN], [grpc_cpp_plugin])
+AS_IF([test "x$GRPC_CPP_PLUGIN" = x], [
+    AC_MSG_ERROR([grpc_cpp_plugin not found])
+])
```

Comment:
If gRPC is a requirement now, I think the README needs to be updated

Reply:
I dropped this change for now and I will address this is issue in a different PR.